### PR TITLE
[CFDS]/Hasan/CFDS-4230/Zero spread tnc

### PR DESCRIPTION
**Pull request by hasan-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/cfd/src/Containers/cfd-password-modal-checkbox.tsx
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-password-modal-checkbox.tsx:1]: Let's reuse the Checkbox of jurisdiction modal

File: packages/cfd/src/Containers/cfd-password-modal.tsx [hunk 1]
```diff
@@ -325,8 +333,22 @@ const CreatePassword = ({
                                 <Localize i18n_default_text='Your MT5 Financial STP account will be opened through Deriv (FX) Ltd. All trading in this account is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA). None of your other accounts, including your Deriv account, is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA).' />
                             </div>
                         )}
+                        {product === PRODUCT.ZEROSPREAD && account_type.category === CATEGORY.REAL && (
+                            <>
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-password-modal.tsx]: <React.Fragment> please

**Review by hirad-deriv — APPROVED** [review]

File: packages/components/src/components/icon/icons.js [hunk 2]
```diff
@@ -446,6 +446,7 @@ import './common/ic-identity-document-verification-verified.svg';
 import './common/ic-identity-document-verification.svg';
 import './common/ic-image.svg';
 import './common/ic-info-blue.svg';
+import './common/ic-info-light.svg';
```
Comment (farrah-deriv) [inline packages/components/src/components/icon/icons.js:449]: is this different from ic-info-blue?

**Review by shontzu-deriv — APPROVED** [review]

File: packages/components/src/components/icon/icons.js (hunk 2 above)
Comment (hasan-deriv) [reply packages/components/src/components/icon/icons.js:449]: yes

File: packages/cfd/src/sass/cfd.scss [hunk 3]
```diff
@@ -174,6 +174,31 @@
...
+    }
+    &__checkbox {
+        display: flex;
+        align-self: baseline;
+        margin-inline-start: -0.8rem;
+        margin-block-end: 3.2rem;
+        .dc-text {
+            font-size: var(--text-size-xxs);
+            text-align: var(--text-align-left);
+        }
```
Comment (farrah-deriv) [inline packages/cfd/src/sass/cfd.scss:194]: can't we handle these as props in `Text` component?

Comment (hasan-deriv) [reply packages/cfd/src/sass/cfd.scss:194]: I am reusing jurisdiction checkbox. All props are already set there. This way is easier.

Comment (farrah-deriv) [reply packages/cfd/src/sass/cfd.scss:194]: We can proceed with this for now but let's refactor it after. Seems like a lot of styles are being overridden.

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**